cypress/page/payment-step.page.ts

```diff
@@ -5,7 +5,7 @@ class PaymentStepPage {
 
     constructor() {
         this.bankwireBtn = ".bankwire";
-        this.confirmOrderBtn = "#cart_navigation > .button > span";
+        this.confirmOrderBtn = "#cart_navigation > .button";
```

Comment:
can you use an attribute to indicate we are referring to the confirmation button?